feature-libs/product-configurator/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.ts

```diff
@@ -127,6 +107,10 @@ export class ConfiguratorAttributeProductCardComponent implements OnInit {
     return isProductCardSelected ? isProductCardSelected : false;
   }
 
+  /**
+   * Computes product price
```

Comment:
Could you elaborate more which price is taken in which case? This is not clear with this description.

Reply:
Good point. I saw that the method is named a bit weirdly, as its purpose is boolean => changed that, adjusted the signature, tests and comments